client/src/app/components/mission-my/mission-my.component.ts

```diff
@@ -11,11 +14,21 @@ import {MissionRouter} from "src/app/router/MissionRouter";
 export class MissionMyComponent implements OnInit {
   public missions = [];
   public roles = [];
+  public languages = Object.keys(Language).map((lang) => Language[lang]);
+  public currentLang: Language = new Locale().get() as Language;
 
-  constructor(private usersServices: UsersService, private router: Router) {}
+  constructor(
```

Comment:
Same as add-vulns-to-host-external.component.ts you don't need to import or inject anything in your component.

Reply:
You can revert the changes for this file